source/_includes/code_examples/code_example.liquid

```diff
@@ -1,11 +1,10 @@
-<div class="sl-c-to-playground">
+<span class="sl-c-to-playground"></span>
```

Comment:
@jamesnw this doesn't work since the two links below need to be in the sl-c-to-playground wrapper for the layout to work (with the current styles).

Reply:
Oh whoops, that was an accidental commit- sorry!

Reply:
updated. @jamesnw let me know if that looks OK to you.